src/Data.php

```diff
@@ -24,19 +24,19 @@ class Data
     public $mappingCleanNameMethodGroup = array();
 
     // 1st array used, stores the csv of country to zone
-    private $mappingCountryToZone = array();
+    public $mappingCountryToZone = array();
```

Comment:
I'm going to leave this in for this commit as I need it for the tests but I will add an issue to add getters and setters for all public arrays